ui/app/selectors/selectors.js

```diff
@@ -475,3 +481,15 @@ export function getIsSwapsChain(state) {
   const chainId = getCurrentChainId(state);
   return ALLOWED_SWAPS_CHAIN_IDS[chainId];
 }
+
+export function getNativeCurrencyImage(state) {
+  if (getIsMainnet(state)) {
```

Comment:
See my previous comment. If we have a `CHAIN_ID_TOKEN_IMAGE_MAP`, which includes the test networks, then we can simplify this logic to:

```
function getNativeCurrencyImage(state) {
    const chainId = getCurrentChainId(state);
    return CHAIN_ID_TOKEN_IMAGE_MAP[chainId];
}
```

This will then allow us to add future symbols without adding more `else if` checks here.

Reply:
Done.